Create resolution-002.md (#345)
hyphacoop/organizing: resolutions/resolution-002.md

# ANNUAL RESOLUTIONS OF THE BOARD OF DIRECTORS

<sup>[From template][template]</sup>

**Resolution(s) in writing** of the board of directors of Hypha Worker Co-operative Inc. (the "Co-operative") 

> from the meeting of the directors held by electronic means on this 13 day of June, 2020.

Author: ```suggestion
from the meeting of the directors held by electronic means on this 13th day of June, 2020.
```

**BACKGROUND:**

1. For the first annual meeting of the members the directors have put forward bylaws and an amendment to the articles of the Co-operative to vary the provisions in its articles to better align its worker and non-share capital status.

2. Two new directors have been elected to the board.

**IT WAS RESOLVED THAT:**

1. The directors of the board unanimously put forward the bylaws for adoption by the membership.

2. The directors of the board unanimously adopt an amendment of the articles for adoption.

> 3. [  ] of the Co-operative is authorized to sign all documents and perform such acts as may be necessary or desirable to give effect to the above resolution.

Author: @dcwalk what is `[  ]` supposed to be?

> 3. [  ] of the Co-operative is authorized to sign all documents and perform such acts as may be necessary or desirable to give effect to the above resolution.

Reviewer: ```suggestion
3. Any two directors or officers of the Co-operative are authorized to sign all documents and perform such acts as may be necessary or desirable to give effect to the above resolution.
```

4. The following individuals are appointed and confirmed as officers of the Co-operative for a term of one year or until replaced:

    - Yurko Ꭻаrеⅿ𝗄ס: President;
    - Patrick Connolly: Vice-President;
    - Udit Vira: Treasurer; and
    - Benedict Lau: Secretary

> Dated in the Province of Ontario on the 13 day of June, 2020.

Author: ```suggestion
Dated in the Province of Ontario on the 13th day of June, 2020.
```
